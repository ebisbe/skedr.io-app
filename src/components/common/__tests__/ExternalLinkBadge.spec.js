import { mount } from '@vue/test-utils'
import Comp from '../ExternalLinkBadge'
import Vue from 'vue'
import Vuetify from 'vuetify'

Vue.use(Vuetify)

describe('ExternalLinkBadge component', () => {
  it('has a snapshot', () => {
    const wrapper = mount(Comp, {
      propsData: {
        hover: false,
        href: 'https://www.flickr.com/groups/groupId'
      },
      slots: { default: 'Default icon text' }
    })
    expect(wrapper).toMatchSnapshot()
  })
})
