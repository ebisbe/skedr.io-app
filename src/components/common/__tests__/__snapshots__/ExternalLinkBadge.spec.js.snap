// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExternalLinkBadge component has a snapshot 1`] = `<span class="v-badge external-link v-badge--overlap"> Default icon text<span class="v-badge__badge grey" style="display: none;"><a href="https://www.flickr.com/groups/groupId" target="_blank" style="text-decoration: none;"><i aria-hidden="true" class="v-icon notranslate material-icons theme--light white--text">open_in_new</i></a></span></span>`;
